test/org/loklak/harvester/RedirectUnshortenerTest.java

```diff
@@ -13,17 +13,17 @@
     @Test
     public void testRedirectUnshortener() {
         HashMap<String, String> shortlinkMap = new HashMap<>();
-        shortlinkMap.put("http://tmblr.co/Z6YPNx1jL1hHK", "http://myjourneymyway.tumblr.com/post/117390619732/first-time-making-loklak-my-grandma-would-be#_=_");
-        shortlinkMap.put("http://dlvr.it/8kTDbJ", "http://islamtimes.org/en/doc/news/442892/?utm_source=dlvr.it&utm_medium=twitter");
```

Comment:
there have been cases that this test failed because of small changes in the websites' url like `http` to `https` . So I and @singhpratyush got urls of some websites(which we assumed) will not make changes in their urls too often. This will let this test fail less often and more reliable. :)